### PR TITLE
Feature/testing snapshots

### LINKED ISSUE #7: Test Landing Page
* This might need integration tests later on, once the other two pages get made

### PR DESCRIPTION
**Pull request by holladayian**
Pull Request Template
### Participating Group Members:
Ian H
 
### Have Tests Been Added?
- [ ] No.
- [X] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Any background context you want to provide?
I could not get Caretaker, Receiver, or ReceiverTimes to pass. I believe this is because of some mocking issue, perhaps with the API.
 
### Any packages installed?
none
 
### Message/Questions for reviewer:
See Background context 
 
### Issues: 
Fixes #7 
Addresses #10 
Addresses #13 

 
### Screenshots (if appropriate): 
* 

 
### Tracking Consistency:
- [X] added appropriate labels
- [ ] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [ ] All new and existing tests passed  
- [X] looked at PR preview to check spelling, syntax, formatting, and completion
 
## Tags (for mentor/alumni code review)
I didn't realize thin until just now, but the testing branch @nware1066 has snapshot coverage in there as well. When I first looked at it I thought it had only cypress files, so I disregarded it, but it should be fair to give credit to Naomi for testing as well.

### CODE REVIEW

File: components/Checkin.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ class Checkin extends React.Component {
         checkedStatus: 'Hold to check in'
       }
     } else {
+      console.log("this.checkinable()", this.checkinable())
```
Comment (holladayian) [inline components/Checkin.js:61]: We can make an issue to go in and remove console.logs and comments

File: components/LandingPage.js [hunk 2]
```diff
@@ -5,19 +5,19 @@ import normalize from 'react-native-normalize';
 
 const LandingPage = ({ navigation }) => {
   return (
-    <View style={styles.container}>
+    <View style={styles.container} testID="lp-view">
```
Comment (holladayian) [inline components/LandingPage.js:8]: This testID is unused. We could use it for unit testing, but I think snapshot will cover everything we'll need for now.

File: components/LandingPage.js [hunk 3]
```diff
@@ -5,19 +5,19 @@ import normalize from 'react-native-normalize';
...
       <Header
         containerStyle={{ backgroundColor: '#483D8B', justifyContent: 'space-around', paddingBottom: 10}}
         centerComponent={{ text: 'Never Alone', style: { color: '#fff', fontSize: normalize(30), fontWeight: "bold" } }}
       />
       <View style={styles.box}>
         <Text style={styles.text}>Choose Your Role:</Text>
-        <TouchableOpacity style={styles.careTakerButton} onPress={() => navigation.navigate('Caretaker')} >
+        <TouchableOpacity style={styles.careTakerButton} testID="lp-caretaker-button" onPress={() => navigation.navigate('Caretaker')} >
```
Comment (holladayian) [inline components/LandingPage.js:15]: This testID (and the one below) are used for unit testing. I think they were two of only a few (maybe only two) unit tests.

**Review by holladayian — COMMENTED** [review]
There's about 122K lines of code that I didn't write. They look like they all involve "coverage". I'm assuming because there's ~122K that we don't need to merge them into main. I don't know if I have it in me to go through them and decide what to keep (Just kidding, I definitely know whether or not I have that in me, and spoilers, I dont).

**Comment by nware1066** [comment]
it's hard to comment on testing. I appreciate the time and effort.

File: components/LandingPage.js (hunk 2 above)
Comment (kathrynljackson) [reply components/LandingPage.js:8]: I think we should take it out if it's not going to be used, but keep it if we will use it later.

File: components/Checkin.js (hunk 1 above)
Comment (kathrynljackson) [reply components/Checkin.js:61]: Done!

File: components/LandingPage.js (hunk 2 above)
Comment (nware1066) [reply components/LandingPage.js:8]: I'm always in favor of removing anything we are not using

**Pull request merged**